File: Python/day12.py
number=int(input("Enter number: "))
sod=0
while number > 0:
    digit = number % 10  
    sod += digit  
    number = number // 10 
    
print("Sum of digits:", sod)


# number = int(input("Enter number: "))
# sod= sum(int(digit) for digit in str(number))
# print("Sum of digits:", sod)

File: Java/day12.java
import java.util.Scanner;

public class SumOfDigits {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter number: ");
        int number = scanner.nextInt();
        int sum = 0;

        while (number > 0) {
            int digit = number % 10;
            sum += digit;
            number = number / 10;
        }

        System.out.println("Sum of digits: " + sum);
        scanner.close();
    }
}